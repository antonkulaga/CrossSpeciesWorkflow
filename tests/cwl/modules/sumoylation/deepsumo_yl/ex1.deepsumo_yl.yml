fastaFile:
  class: File
  location: ../data/ex1.fasta
threshold: 0.5
outputFilename: ex1.deepsumo_yl.out
  