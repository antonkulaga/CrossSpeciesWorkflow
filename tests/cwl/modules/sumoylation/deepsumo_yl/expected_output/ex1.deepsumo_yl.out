Fragments name,Sequence,Probability value
>sp|B7VLR2|K15,OOMSLHLSNVILHQLSKNDQEELIVNYRAESLE,0.64316129684
>sp|B7VLR2|K95,YDFSQKSALRLKDELSKYPFADEGTLVLAEYQS,0.51433652639
>sp|B7VLR2|K129,ATDYLFIGLLPSNQSLKVTEGLDISATDYLDIS,0.82540392876
>sp|B7VLR2|K146,VTEGLDISATDYLDISKMDIVARLDLSTYDTDK,0.88024085760
>sp|B7VLR2|K196,ADFFLDFLQAEVGLDAKQQNQVLMQAVEDFVSD,0.51994830370
>sp|B7VLR2|K224,DFVSDSKLEKEEAISYKKQVADYCNEQLKAGDE,0.53503704071
>sp|B7VLR2|K333,TIKGTPPNLRDQLTRNKSOOOOOOOOOOOOOOO,0.59277623892
>sp|P55769|K9,OOOOOOOOMTEADVNPKAYPLADAHLTKKLLDL,0.56262773275
>sp|P55769|K20,ADVNPKAYPLADAHLTKKLLDLVQQSCNYKQLR,0.72789984941
>sp|P55769|K21,DVNPKAYPLADAHLTKKLLDLVQQSCNYKQLRK,0.50280719995
>sp|P55769|K33,HLTKKLLDLVQQSCNYKQLRKGANEATKTLNRG,0.53044354916
>sp|P55769|K76,AEPLEIILHLPLLCEDKNVPYVFVRSKQALGRA,0.55890333652
>sp|P55769|K107,RACGVSRPVIACSVTIKEGSQLKQQIQSIQQSI,0.50342035294
